### PR TITLE
Add examples for alternate solution scoring in GPSR and EEGPSR

### PR DESCRIPTION
**Pull request by kyordhel**
This patch addresses #434 and specifies in the GPSR and EEGPSR appendices how work arrounds can be evaluated by a referee, centering attention in manipulation bypassing for SSPL.

> Bring me a coke

- **Skipping command:** The robot says "I understood you want me to bring you a coke, but I cannot grasp objects, so I'll skip this command". Since the robot is not executing the task, no score is given.
- **Continue Rule:** The robot more or less reaches the position, fuzzily points at the object, and then requests to a human assistant to deliver the coke for it. In this case, the referee might grant up to 1/3 of the points, if any.
- **Requesting human assistance:** Taking advantage of the Continue Rule, the robot requests assistance from a human to grasp the object, requesting later to follow it. During the guiding phase, the robot actively tracks the human to the operator's position and supervises the delivery (e.g. telling it noticed the operator has received the coke). In this case, and regarding the execution of the tasks, the referee may grant a full score.
- **Social alternative:** The robot looks for another person in the arena, finds them, and convinces them (or socially bribes them) to deliver a coke to the operator using natural language dialogs. In these rare cases, the referee would grant full score and even the team might request a bonus for outstanding performance.

### CODE REVIEW

File: tests/GPSR_appendix.tex [hunk 1]
```diff
@@ -87,7 +87,28 @@ \subsubsection{Other tasks [All Leagues]}
...
+
+	\item \textbf{Continue Rule:} The robot more or less reaches the position, fuzzily points at the object, and then requests to a human assistant to deliver the coke for it. In this case, the referee might grant up to $\frac{1}{3}$ of the points, if any.
+
+	\item \textbf{Requesting human assistance:} Taking advantage of the Continue Rule, the robot requests assistance from a human to grasp the object, requesting later to follow it. During the guiding phase, the robot actively tracks the human to the operator's position and supervises the delivery (e.g. telling it noticed the operator has received the coke). In this case, and regarding the execution of the tasks, the referee may grant a full score.
+
+	\item \textbf{Social alternative:} The robot looks for another person in the arena, finds them, and convinces them (or socially bribes them) to deliver a coke to the operator using natural language dialogs. In these rare cases, the referee would grant full score and even the team might request a bonus for outstanding performance.
```
Comment (balkce) [inline tests/GPSR_appendix.tex]: change
"the referee would grant full score and even the team might request a bonus for outstanding performance"

to
"the referee may grant a full score depending on the success of the interaction"

I wouldn't go as far as providing bonuses for a continue rule.

Comment (kyordhel) [reply tests/GPSR_appendix.tex]: I agree with you both @justinhart and @balkce, that's why the update proceeds and now *the referee may grant a full score*. In my opinion, grasping an object is as complex as supervising a grasp, guiding a person, and supervising a delivery which requires a lot of perception. Maybe even less complex regarding the nature of the object.

In any case, the referee can mark the scoresheet or make some notes and discuss it later with the TC prior to assign a score. We have had such situations in the past.

Comment (justinhart) [reply tests/GPSR_appendix.tex]: Fair.

**Comment by justinhart** [comment]
I forget. It's cool if I merge once I approve the PR, right?

**Comment by kyordhel** [comment]
If you approve, you can also merge, unless someone else is requesting changes that haven't been fixed. All reviewers should agree.

You shouldn't approve if you contributed (push something into the PR).

**Comment by justinhart** [comment]
Merged.

**Pull request merged**